css/contact.css

```diff
@@ -0,0 +1,142 @@
+/*
+===================================
+===================================
+Styles for an individual page
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* contact form */
+.wrapper {
+	width: var(--mobile-wrapper-width);
+}
+.form-input {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	/* padding: 20px; */
+}
+input[type="text"],
+[type="email"],
+select,
+textarea {
+	resize: vertical;
+	font-family: var(--main-font);
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	color: var(--color-onyx);
+	background-color: var(--color-white);
+	border: 1px solid #6c6c6c;
+	width: 100%;
+	border-radius: 5px;
+	padding: 10px;
+	margin: 5px 0;
+}
+.message {
+	min-height: 140px;
+	width: 100%;
+	border-radius: 10px;
+	border: none;
+	margin: 10px 0;
+	align-self: flex-start;
+}
+.button2 {
```

Comment:
It looks like the font family is not being applied to the "Submit" button. 

<img width="366" alt="Screen Shot 2024-01-11 at 1 23 00 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/8c83e466-f383-43e4-93ec-f1774ab7581e">